test/testdrive/source-timestamps.td

```diff
@@ -59,9 +59,6 @@ ALTER SYSTEM SET kafka_default_metadata_fetch_interval = '1s'
 ALTER SYSTEM SET pg_offset_known_interval = '1s'
 ALTER SYSTEM SET mysql_offset_known_interval = '1s'
 
-> ALTER CLUSTER test SET (REPLICATION FACTOR 0)
```

Comment:
mostly just curious: why are these removed?

Reply:
They existed because applying the changed intervals needed a source restart before this PR, but it doesn't need one now!

Reply:
:man_facepalming: